### ISSUE TITLE
Pre-trained final model 

### ISSUE THREAD

**Issue by realEbi** [plkmo/BERT-Relation-Extraction#40]
Hi
Thanks for the brilliant implementation.
Do you or anyone from the community have trained the model and can give access for downloading for everyone usage?
I don't want to train an already trained model, it's not healthy for the environment!
Thanks.

**Comment by plkmo** [comment]
Yes if anyone could share a pre-trained model on a large corpus that would be great. I could only pre-train on a small CNN corpus as a proof-of-concept. 
Will leave this open for anyone to share their pre-trained models
